Source/JavaWrappers.cpp

```diff
@@ -274,6 +282,112 @@ namespace java::lang
     }
 }
 
+namespace java::websocket
```

Comment:
Don't you need a Destructor?

Reply:
I already have a destructor: https://github.com/BabylonJS/AndroidExtensions/pull/2/files/ac0b3238ebbf73481ea623693581b665caf6664e#:~:text=WebSocketClient%3A%3A~WebSocketClient()